pymbolic/primitives.py

```diff
@@ -186,7 +214,7 @@ def disable_subscript_by_getitem():
     pass
 
 
-class Expression(ABC):
+class Expression:
```

Comment:
Does `abstractmethod` do anything without the `ABC` metaclass?

Reply:
Nope, removed.